src/components/Graphs/GaugeGraph/index.js

```diff
@@ -0,0 +1,142 @@
+import React from "react";
+
+import AbstractGraph from "../AbstractGraph";
+
+import * as d3 from "d3";
+
+import "./style.css";
+
+import {properties} from "./default.config"
+
+
+export default class GaugeGraph extends AbstractGraph {
+
+    constructor(props) {
+        super(props, properties);
+    }
+
+    deg2rad(deg) {
+        return deg * Math.PI / 180;
+    }
+
+    render() {
+        const {
+            data,
+            width,
+            height
+        } = this.props;
+
+        if (!data || !data.length)
+            return;
+
+        const {
+            minColumn,
+            maxColumn,
+            currentColumn,
+            margin,
+            gaugePtrWidth,
+            gaugePtrTailLength,
+            gaugePtrHeadLengthPercent,
+            gaugeLabelInset,
+            gaugePtrColor,
+            gaugeTicks,
+            gaugeRingInset,
+            gaugeRingWidth,
+            labelFormat,
+            fontColor,
+            colors
+        } = this.getConfiguredProperties();
+
+        const angles = {
+            min: -90,
+            max: 90
+        };
+
+        let availableWidth     = width - (margin.left + margin.right);
+        let availableHeight    = height - (margin.top + margin.bottom);
+
+        const minValue         = data[minColumn] ? data[minColumn] : 0;
+        const maxValue         = data[maxColumn] ? data[maxColumn] : 100;
+        const currentValue     = data[currentColumn] ? data[currentColumn] : 20;
+
+        const minRadius   = Math.min(availableWidth, availableHeight) / 2;
+
+        const innerRadius = minRadius - gaugeRingInset - gaugeRingWidth;
+        const outerRadius = minRadius - gaugeRingInset;
+
+        const pointerHeadLength = Math.round(minRadius * gaugePtrHeadLengthPercent);
+
+        const arcColor = d3.interpolateHsl(d3.rgb(colors[0] ? colors[0] : '#e8e2ca'), d3.rgb(colors[1] ? colors[1] : '#3e6c0a'))
+
+        const range = angles.max - angles.min;
+
+        const arc = d3.arc()
+    			  .innerRadius(innerRadius)
```

Comment:
There's a spacing issue here with tabs vs. spaces. Please use spaces rather than tabs, to be consistent with the rest of the codebase (you can see this if you look at the [diff](https://github.com/nuagenetworks/visualization-framework/pull/359/files)). Thanks!